### ISSUE TITLE
Latest repository release doesn't show up on PyPI

### ISSUE THREAD

**Issue by Carifio24** [glue-viz/glue-qt#23]
Our latest released version here on GitHub is `0.3.2`, but the newest version that I see on PyPI is `0.3.1`. I guess the release CI failed for our last release?

**Comment by astrofrog** [comment]
Yes this is due to all the CI being broken with the latest pytest. I'll see what I can do tomorrow.
